rest-api/edit-conversation-image-title/styles.css

```diff
@@ -0,0 +1,60 @@
+.modal {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+    font-family: 'Roboto', sans-serif;
```

Comment:
Probably unnecessary? Is there a reason our example specifically should be using roboto?